### ISSUE TITLE
Telescope use wrong window to show results

### ISSUE THREAD

**Issue by ribelo** [nvim-telescope/telescope.nvim#1950]
### Description

as in the title

### Neovim version

```markdown
NVIM v0.8.0-dev
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by nixbld
```

### Operating system and version

nixos-unstable

### checkhealth telescope

```markdown
telescope: require("telescope.health").check()
========================================================================
## Checking for required plugins
  - OK: plenary installed.
  - OK: nvim-treesitter installed.

## Checking external dependencies
  - OK: rg: found ripgrep 13.0.0
  - OK: fd: found fd 8.3.2

## ===== Installed extensions =====

## Telescope Extension: `file_browser`
  - INFO: No healthcheck provided

## Telescope Extension: `frecency`
  - INFO: No healthcheck provided

## Telescope Extension: `fzy_native`
  - INFO: No healthcheck provided

## Telescope Extension: `lazygit`
  - INFO: No healthcheck provided

## Telescope Extension: `projects`
  - INFO: No healthcheck provided
```

### Steps to reproduce

1. open two window with different files
2. switch to second window
3. `:Telescope current_buffer_fuzzy_find`, search something `<CR>`
4. the result will appear in the first window instead of the second

### Expected behavior

result should be shown in the window from which the command was called

### Actual behavior

the result is always shown in the first window

### Minimal config

```Lua
irrelevant
```

**Comment by ribelo** [comment]
I was 3 minutes late... #1949

**Comment by fdschmidt93** [comment]
Apologies :) should be fixed now with #1951
